customers-service/src/main/java/com/petclinic/customers/datalayer/PhotoRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.petclinic.customers.datalayer;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface PhotoRepository extends JpaRepository<Photo, String> {
+    Photo findPhotoById(int id);
+    Photo findPhotoByName(String name);
+    Boolean existsById(int id);
```

Comment:
Not urgent, when possible, please explain what methods/classes require this data?

Reply:
set and get photo use the findPhoto, while the exists is only use to overwrite the default one as it need this method in testing. We use name as an index and thus findById or existsById will expect a String by default.